src/lib/stores/workstreamsStore.js

```diff
@@ -1,13 +1,13 @@
 import { writable } from 'svelte/store';
 
-export const workstreams = writable([]);
+export const workstreamStore = writable([]);
```

Comment:
@deckardkrug naming the store `workstreamStore` helps a lot with readability imo.

Reply:
Okay, I will make that change.